lib/lint.js

```diff
@@ -15,9 +16,21 @@ const internals = {
 exports.lint = function (settings, callback) {
 
     const linterPath = (settings.linter && settings.linter !== 'eslint') ? settings.linter : internals.linter;
-    const child = ChildProcess.fork(linterPath,
-                                  settings['lint-options'] ? [settings['lint-options']] : [],
-                                  { cwd: settings.lintingPath });
+
+    let linterOptions;
+
+    try {
+        linterOptions = JSON.parse(settings['lint-options']);
+    }
+    catch (err) {
+        linterOptions = {};
```

Comment:
In this case we should throw with an error informing the user that `lint-options` couldn't be parsed.